src/test/regress/expected/views_create.out

```diff
@@ -0,0 +1,63 @@
+CREATE SCHEMA views_create;
+SET search_path TO views_create;
+CREATE TABLE view_test_table(a INT NOT NULL PRIMARY KEY, b BIGINT, c text);
+CREATE OR REPLACE VIEW select_filtered_view AS
+    SELECT * FROM view_test_table WHERE c = 'testing'
+    WITH CASCADED CHECK OPTION;
+CREATE OR REPLACE VIEW select_all_view AS
+    SELECT * FROM view_test_table
+    WITH LOCAL CHECK OPTION;
+CREATE OR REPLACE VIEW count_view AS
+    SELECT COUNT(*) FROM view_test_table;
+SELECT create_distributed_table('view_test_table', 'a');
+ create_distributed_table
+---------------------------------------------------------------------
+
+(1 row)
+
+INSERT INTO view_test_table VALUES (1,1,'testing'), (2,1,'views');
+SELECT * FROM count_view;
+ count
+---------------------------------------------------------------------
+     2
+(1 row)
+
+SELECT COUNT(*) FROM count_view;
+ count
+---------------------------------------------------------------------
+     1
+(1 row)
+
+SELECT COUNT(*) FROM select_all_view;
```

Comment:
shouldn't we move these queries to non-create file/schedule? Because queries can be executed from worker nodes, and the arbitrary configs pick a random node to execute queries, which is useful

Reply:
(for non-mx cluster, these queries executed from the coordinator)

Reply:
Ah, we cannot do that because views are not distributed to the workers now. 

cc: @velioglu , once we do that, we can split this file